src/FastRoute/src/DBWrapper.cpp

```diff
@@ -1218,13 +1206,13 @@ int DBWrapper::checkAntennaViolations(const std::vector<FastRoute::NET>* routing
     }
     wireEncoder.end();
 
-    odb::orderWires(dbNets[netName], false, false);
+    odb::orderWires(_dbNets[netName], false, false);
 
     std::vector<VINFO> netViol
-        = arc->get_net_antenna_violations(dbNets[netName]);
+        = arc->get_net_antenna_violations(_dbNets[netName]);
     if (netViol.size() > 0) {
-      _antennaViolations[dbNets[netName]->getConstName()] = netViol;
-      _dirtyNets.push_back(dbNets[netName]);
+      _antennaViolations[_dbNets[netName]->getConstName()] = netViol;
```

Comment:
how is _dbNets[netName]->getConstName() not just netName???
you lookup the net with it's name and get it's name.